onnxruntime/core/optimizer/qdq_transformer/weight_bias_quantization.cc

```diff
@@ -128,6 +129,14 @@ Status WeightBiasQuantization::ApplyImpl(Graph& graph, bool& modified, int graph
       int64_t axis = 1;
       if (auto axis_iter = dq_attrs.find("axis"); axis_iter != dq_attrs.end()) {
         axis = axis_iter->second.i();
+        const ONNX_NAMESPACE::TensorShapeProto* weight_shape = weight_arg->Shape();
+        if (!weight_shape && dq_1 && dq_1->InputDefs()[0]) {
```

Comment:
nit: it is redundant to check for `dq_1 != nullptr` here as that ensured earlier.

https://github.com/CodeLinaro/onnxruntime/blob/01112314cc770c15db297b8a37b31ea9110dcb04/onnxruntime/core/optimizer/qdq_transformer/weight_bias_quantization.cc#L114